doc/ref_arch/openstack/chapters/chapter08.md

```diff
@@ -5,7 +5,28 @@
 
 ## Table of Contents
 * [8.1 Introduction](#8.1)
-* [8.2 Heading](#8.2)
+
+The purpose of this chapter is to identify the gaps between what is required for automated deployment of VNFs on NFVI frameworks and framework offered by OpenStack.Once gaps are identified, the next step will be to propose a plan to address these gaps. 
```

Comment:
As per @ASawwaf Gaps will be wider than just OpenStack. can we re-phrase to be more generic. i.e. it is not just about automation the deployment. it could include more than just that and impact more than just OpenStack.